### DISCUSSION TITLE
New Maintainer?

### DISCUSSION THREAD

**Discussion by smorks** [smorks/keepassnatmsg#104] [Announcements]
as mentioned here: https://github.com/smorks/keepassnatmsg/pull/102#issuecomment-1343031223

i don't actually use this plugin anymore, therefore i don't have the drive or time to maintain this project. i plan on merging in the current two active pull requests (#101, #102), and creating a new release.

beyond that, does anybody want to take over at the maintainer for this project? i can help with any questions about the current codebase, etc.

thanks!

**Comment by Silvenga** [comment]
I completely depend on KeepassNatMsg, I use it everyday, so I would hate to see it to succumb to entropy overtime. I likely could maintain/help maintain it, I have [some OSS experience](https://github.com/Silvenga) and I'm a full time .NET developer. Unfortunately, my experience with KeePass plugins is somewhat limited outside of fixing small bugs - so I'm definitely far from an ideal candidate.

**Reply by smorks** [reply]
sorry for the super late response. any help would be appreciated at this point. i can add you to this repo if you're still interested, and feel free to ask me any questions about the codebase, etc.

**Reply by Silvenga** [reply]
I would be interested!

Is there a feature/bug that might be a candidate as a "good first ticket"? Just to get me exposed.

**Reply by smorks** [reply]
#118 might be good yo get to know how messages are passed between the extension and the plugin.

#88 would be mostly plugin work I think.

and #71 for learning how the plugin communicates with the extension maybe.

I'll add you to the repo now.

**Reply by Silvenga** [reply]
Awesome, thanks!

**Reply by Silvenga** [reply]
I'll take a crack at #88 - I also have the same issue (I also swear it used to work).

**Reply by smorks** [reply]
not sure if this will help or if it's something you already know, but for easier debugging, you can compile the plugin in debug, then put in the KeePass's plugins folder, run KeePass, and then use the "Debug -> Attach to Process" option in Visual Studio to make debugging a little easier. Using that should make breakpoints, etc, work.

**Reply by Silvenga** [reply]
Always a help, thanks.

**Comment by EnigmaSolved** [comment]
Though I am probably not able to contribute code, I also use KeepassNatMsg daily and am extremely grateful to see @Silvenga being willing to step in as maintainer. 😊

**Reply by Silvenga** [reply]
Thanks - I appreciate the support. Although, I have to say, RL has been kicking my butt over the past month - hopefully I'll have time to dive-in, in the next couple of weeks.

**Comment by mcarbonneaux** [comment]
> i don't actually use this plugin anymore, therefore i don't have the drive or time to maintain this project. i plan on merging in the current two active pull requests (#101, #102), and creating a new release.

what do use in place ?

**Reply by smorks** [reply]
> what do use in place ?

i'm using [vaultwarden](https://github.com/dani-garcia/vaultwarden) currently. just easier sharing with family, etc.

**Reply by smorks** [reply]
also, if you prefer a maintained cloudless solution, i would recommend going with [KeePassXC](https://keepassxc.org/) itself.

**Reply by mcarbonneaux** [reply]
the bitwarden client support the equivalent of this module ? (autofill in browser)

**Reply by smorks** [reply]
> the bitwarden client support the equivalent of this module ? (autofill in browser)

yes, it does, works quite well IMO.

**Reply by jonjonez** [reply]
Hello? Is this plugin still being maintained? Is it still safe to use? I wouldn't want to have to switch to another one, since it's the only one that communicates with keepassxc-browser.

**Reply by EnigmaSolved** [reply]
@jonjonez I can't speak to the maintenance question, but I am using this plugin daily (with KeePass) for a year or so and it works great!

**Reply by smorks** [reply]
> Hello? Is this plugin still being maintained? Is it still safe to use? I wouldn't want to have to switch to another one, since it's the only one that communicates with keepassxc-browser.

Some people have offered to help but so far there hasn't been any movement, AFAIK.

Glad to hear it's still working and I can always do bug fixes when i have time too.

**Reply by mcarbonneaux** [reply]
the problem with bitwarden is not supporting field reference...

**Comment by gardian12** [comment]
Hello @smorks,

I wanted to reach out regarding the maintainer topic, which has been open for four years now. I would like to offer my support or even take over the role.

I've been using the plugin daily for years, and it has consistently worked without any issues. Additionally, updates on KeePass and the browser plugin have not caused any problems.

I also contributed to the repository a while back when the only error I was aware of occurred.
Also at KeePass I published plugins.

**Reply by smorks** [reply]
sure, I'll add you as a collaborator.

**Reply by gardian12** [reply]
Hello @smorks,
I'm sorry I missed your invitation to collaborate. Since the invitation has timed out, could you please send it again?
Thank you very much!

**Reply by gardian12** [reply]
Hello @smorks,

Would it be possible for me to be added as a contributor to the [keepassnatmsg-proxy](https://github.com/smorks/keepassnatmsg-proxy) project as well? I believe my contributions could be valuable.

Thank you!

**Reply by smorks** [reply]
sorry for the delay, I've invited you again to both repo's.

**Reply by gardian12** [reply]
Many thanks, @smorks
